bitnami/mongodb-sharded/templates/shard/shard-arbiter-statefulset.yaml

```
@@ -44,7 +44,7 @@ spec:
       hostAliases: {{- include "common.tplvalues.render" (dict "value" $.Values.shardsvr.arbiter.hostAliases "context" $) | nindent 8 }}
       {{- end }}
       {{- if $.Values.shardsvr.arbiter.affinity }}
-      affinity: {{- include "common.tplvalues.render" (dict "value" $.Values.shardsvr.arbiter.affinity "context" $) | nindent 8 }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" $.Values.shardsvr.arbiter.affinity "context" (set $ "dataNodeLoopId" $i)) | nindent 8 }}
```

Given that you will iterate over arbiters instead of data nodes, what if we change the name of the local variable?
```suggestion
      affinity: {{- include "common.tplvalues.render" (dict "value" $.Values.shardsvr.arbiter.affinity "context" (set $ "arbiterLoopId" $i)) | nindent 8 }}
```